pkgs/development/embedded/teensy-loader-cli/default.nix

```
@@ -6,15 +6,15 @@
 , libusb-compat-0_1
 }:
 
-stdenv.mkDerivation rec {
+ stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation {
```
we don't need to use finalAttrs here

---

Then how should I reuse the version attribute for `src.rev`?